tools/oio-travis-tests.sh

```diff
@@ -270,6 +270,12 @@ if is_running_test_suite "variables" ; then
 	tox -e variables
 fi
 
+if is_running_test_suite "sdk" ; then
```

Comment:
IMO this is never called as we have a `exit 0` just after `make test` with `SDK` mode in `.travis.yml`